clients/privacy-center/cypress/e2e/consent-banner.cy.ts

```diff
@@ -2540,8 +2540,8 @@ describe("Consent overlay", () => {
               id: "pri_notice-applied",
               notice_key: "applied",
               has_gpc_flag: true,
-              consent_mechanism: ConsentMechanism.OPT_OUT,
-              default_preference: UserConsentPreference.OPT_IN,
+              consent_mechanism: ConsentMechanism.OPT_IN,
+              default_preference: UserConsentPreference.OPT_OUT,
```

Comment:
This was a bug. 

If GPCis enabled for the notice and consent is opt-out, we apply the "APPLIED" badge. 

If GPC is enabled for the notice and consent is opt-in, we apply the "OVERRIDDEN" badge.